app/services/TariffApiService.scala

```diff
@@ -91,3 +102,13 @@ class TariffApiService @Inject()(tariffApiConfig: TariffApiConfig, tariffApiConn
       case JsError(_)          => None
     }
 }
+
+object TariffApiService {
+
+  sealed trait NoInfoForCommodity
+
+  case object SkipSupplementaryUnitsPage extends NoInfoForCommodity
+  case object DisplayYesNoSupplementaryUnitsPage extends NoInfoForCommodity
```

Comment:
Nit: I understand their intent but these object names seem a little odd in this layer. The service layer should not know/concern itself with what UI pages exist. Maybe `DisplayYesNoSupplementaryUnitsPage` => 'TariffApiUnavailable' & `SkipSupplementaryUnitsPage`=> 'SupplimentaryUnitsNotRequired' ?